kmm/car_direction.py

```diff
@@ -0,0 +1,6 @@
+from enum import Enum
+
+
+class CarDirection(Enum):
```

Comment:
```
class CarDirection(Enum, str):
```
Not sure what the difference is?

Reply:
Is that common practice? I dont really know what happens when you inherit from `str`